about.html

```diff
@@ -0,0 +1,189 @@
+      </div>
+
+      <hr class="about__line">
+      <!-- speakers -->
+      <div class="events">
+        <h2 class="program__header">
+          <span>See Past Events</span>
+          <span class="border-line">&nbsp;</span>
+        </h2>
+        <div class="events__wrapper">
+          <div class="events__one">
+            <div class="events__bg-image">
+              <h2>2019</h2>
+              <p>Women In Tech Africa Summit 2019</p>
+            </div>
+          </div>
+          <div class="events__one">
+            <div class="events__bg-image">
+              <h2>2019</h2>
+              <p>Women In Tech Africa Summit 2019</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- partner -->
+      <div class="partner" id="about-partner">
+        <h2 class="program__header">
+          <span class="partner__header">Partner</span>
+          <span class="border-line">&nbsp;</span>
+        </h2>
+        <div class="partner__wrapper">
+          <svg class="partner__icon">
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-google-plus"></use>
+          </svg>
+          <svg class="partner__icon">
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-facebook1"></use>
+          </svg>
+          <svg class="partner__icon">
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-twitter"></use>
+          </svg>
+          <svg class="partner__icon">
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-npm"></use>
+          </svg>
+          <svg class="partner__icon">
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-amazon
+            "></use>
+          </svg>
+          <svg class="partner__icon">
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-gnome"></use>
+          </svg>
+          <svg class="partner__icon">
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-mozilla"></use>
+          </svg>
+          <svg class="partner__icon">
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-airbnb"></use>
+          </svg>
+          <svg class="partner__icon">
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-github"></use>
+          </svg>
+          <svg class="partner__icon">
+            <use xlink:href="./assets/icons/symbol-defs.svg#icon-microsoft"></use>
+          </svg>
+        </div>
+      </div>
+
+      <footer class="footer" id="about-footer">
+        <div class="legal">
+          &copy;2022. All rights reserved.
+      </div>
+      </footer>
```

Comment:
- [ ] Your footer should contain a logo aligned to the left and text aligned to the right. You can also add this footer to your index.html page even though the main design does not have a footer in the index.html page 👍🏿